spec.html

```diff
@@ -37133,12 +37133,14 @@ <h1>The SharedArrayBuffer Constructor</h1>
       <p>The SharedArrayBuffer constructor:</p>
       <ul>
         <li>is the intrinsic object <dfn>%SharedArrayBuffer%</dfn>.</li>
-        <li>is the initial value of the *"SharedArrayBuffer"* property of the global object.</li>
+        <li>is the initial value of the *"SharedArrayBuffer"* property of the global object, if that property is present (see below).</li>
         <li>creates and initializes a new SharedArrayBuffer object when called as a constructor.</li>
         <li>is not intended to be called as a function and will throw an exception when called in that manner.</li>
         <li>is designed to be subclassable. It may be used as the value of an `extends` clause of a class definition. Subclass constructors that intend to inherit the specified `SharedArrayBuffer` behaviour must include a `super` call to the `SharedArrayBuffer` constructor to create and initialize subclass instances with the internal state necessary to support the `SharedArrayBuffer.prototype` built-in methods.</li>
       </ul>
 
+      <p>If a host does not provide concurrent access to SharedArrayBuffer objects it may omit the *"SharedArrayBuffer"* property of the global object.</p>
```

Comment:
HTML violates this, right? It's a single host that does allow concurrent access to SAB objects. It does vary between its agent clusters, but they're all part of the same host.

Reply:
The way I was looking at it was that HTML only sometimes provides concurrent access to SAB objects. In the cases where it does provide concurrent it must include the SharedArrayBuffer global, and in the cases where it does not provide concurrent access it may omit it.

Would it help to say "whenever" rather than "if"?

Reply:
Can we make it explicit and say "If a host does not provide concurrent access to SAB objects within an agent cluster, ..."?

Although even then we might not want to consistently hide it within the entire cluster. We decided to do that for now, but I personally would like to keep the door open to still exposing it in worklets and workers and eventually everywhere again (it's clear that everywhere is allowed though).

Reply:
I think @annevk's point is valid, and I also think "whenever" + "may" suffices to convey enough latitude without extra wording about where exactly that property may be not present.

Reply:
Changed to "whenever".